crypto-ffi/bindings/js/test/wdio/proteus.test.ts

```diff
@@ -21,9 +21,8 @@ describe("proteus", () => {
         const alice = crypto.randomUUID();
         await proteusInit(alice);
         const result = await browser.execute(async (clientName) => {
-            const lastResortPrekeyId =
-                window.ccModule.CoreCrypto.proteusLastResortPrekeyId();
```

Comment:
I don't think we should make this change; last resort prekey _should_ be static.